src/browser/modules/Stream/Visualization.jsx

```diff
@@ -84,7 +102,31 @@ export class Visualization extends Component {
             reject({nodes: [], rels: []})
           } else {
             let count = response.result.records.length > 0 ? parseInt(response.result.records[0].get('c').toString()) : 0
-            resolve({...bolt.extractNodesAndRelationshipsFromRecordsForOldVis(response.result.records, false), count: count})
+            const graph = bolt.extractNodesAndRelationshipsFromRecordsForOldVis(response.result.records, false)
+            this.autoCompleteRelationships(this.state.nodesAndRelationships.nodes, graph.nodes)
+            resolve({...graph, count: count})
+          }
+        }
+      )
+    })
+  }
+
+  getInternalRelationships (existingNodeIds, newNodeIds) {
+    existingNodeIds = existingNodeIds.concat(newNodeIds)
+    const query = `
+            MATCH (a)-[r]-(b) WHERE id(a) IN {existingNodeIds}
```

Comment:
The "new" syntax for params is `$existingNodeIds` instead of `{existingNodeIds}`.
We should probably use that.